src/domain/transfer/transformer.js

```diff
@@ -62,15 +74,40 @@ const transformHeaders = (headers) => {
         }
         normalizedHeaders[headerKey] = tempDate
         break
-      case ('content-length'):
+      case (ENUM.headers.GENERAL.CONTENT_LENGTH.toLowerCase()):
         // Do nothing here, do not map. This will be inserted correctly by the Hapi framework.
         break
+      case (ENUM.headers.FSPIOP.URI.toLowerCase()):
+        // Do nothing here, do not map. This will be removed from the callback request.
+        break
+      case (ENUM.headers.FSPIOP.HTTPMETHOD.toLowerCase()):
```

Comment:
Please, change to HTTP_METHOD. I'm also not sure about .toLowerCase() part of the *case* statement, since enums are under our control, compared to *switch* headerKey.toLowerCase.